presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -239,6 +241,10 @@ protected void setup(Binder binder)
             config.setKeyStorePath(internalCommunicationConfig.getKeyStorePath());
             config.setKeyStorePassword(internalCommunicationConfig.getKeyStorePassword());
         });
+        if (internalCommunicationConfig.getInternalLdapCommunicationUser() != null) {
```

Comment:
This can be moved to https://github.com/prestodb/presto/pull/9683/commits/edca72763676675777adbf99aec02e6b8ed9ced7 and refactored once the Module is merged